streams/src/main/java/org/apache/kafka/streams/processor/TaskMetadata.java

```diff
@@ -27,10 +27,12 @@
 
 /**
  * Represents the state of a single task running within a {@link KafkaStreams} application.
+ * @deprecated since 3.0, not intended for public use, use {@link org.apache.kafka.streams.TaskMetadata} instead.
```

Comment:
```suggestion
 * @deprecated since 3.0, use {@link org.apache.kafka.streams.TaskMetadata} instead.
```
I think people might be a bit confused if we say this, just point them to the new interface